src/test/java/teammates/test/cases/browsertests/InstructorCoursesPageUiTest.java

```diff
@@ -279,13 +283,49 @@ private void testSortCourses() {
         coursesPage.sortByCourseId().verifyTablePattern(0, patternString);
     }
 
-    private void testDeleteAction() throws Exception {
+    private void testMoveToRecycleBinAction() throws Exception {
```

Comment:
Try to rearrange the lines in a test to reflect a given-when-then structure using newlines. In this case, the `given` refers to the definition of `courseId` and the pre-assertion. The `when` is when `coursesPage.moveCourseToRecovery(courseId)` is called, then the `then` part is the post-assertions/verifications. We should have 3 groups of lines separated by newline in this case.

This is a minor change (not real changes, actually, since it's just arranging the existing code), but it makes parsing a test easier when someone other than you has to maintain it 😄 

General idea:
https://blog.codecentric.de/en/2017/09/given-when-then-in-junit-tests/

Reply:
Thanks for sharing! Yeah that really helps :)